### ISSUE TITLE
golang casting in byte operations?

### ISSUE THREAD

**Issue by AlexMouton** [andyarvanitis/purescript-native#36]
Hi Andy
Found this code generated in Data_Int while goofing with Random this evening.
Looks like x should be x.(int)?

Thanks
Alex

```
func PS__even() Any {
    return func(x Any) Any {
        return (x & 1) == 0
    }
}
```

**Comment by andyarvanitis** [comment]
Yup, I spotted that bug recently*. If you pull the latest version it will have the fix for it. Note that I made some changes to the FFI recently too, so if you wrote any of your own, you'll want to look at how the ones in the latest version of `purescript-native-go-ffi` does things.

*Specifically, fixed in f7489a219d0f137e665672cb2768e4c88f669f54

**Comment by andyarvanitis** [comment]
I'm going to go ahead and close this since it was fixed over a week ago (https://github.com/andyarvanitis/purescript-native/commit/f7489a219d0f137e665672cb2768e4c88f669f54). Please let me know if you still see it with the latest version.
